lib/game.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at, :hash
+
+  def initialize(author, publish_date, last_played_at, multiplayer)
+    super(publish_date)
+    @last_played_at = Date.strptime(last_played_at)
+    @multiplayer = multiplayer
+    add_author(author)
+  end
+
+  def to_hash
+    {
+      'author' => {
+        'first_name' => author.first_name,
+        'last_name' => author.last_name
+      },
+      'publish_date' => publish_date.to_s,
+      'last_played_at' => last_played_at,
+      'multiplayer' => multiplayer
+    }
+  end
+
+  private
+
+  def can_be_archived?
+    super && (Date.today - last_played_at > 2)
+  end
```

Comment:
- [ ] Let's check this together :smile: 
- **Hints**
- recent_year = Date.today.year 
- 2_years_ago = Date.new(recent_year -2)
- Should return true if parent's method return true AND if `last_played_at` is older than 2 years ie `super && last_played_at < 2_years_ago
`

Reply:
@FrederickMih , kindly clear my doubt is'nt it should be `super && last_played_at > 2_years_ago` because we are checking if `last_played_at` is greater then 2 years or not

Reply:
@devendra-alt, in the requirements it says if last_play_at is `older ` than 2 years ago. According to my understanding, older than is `less tha`n :smile:

Reply:
Hello @FrederickMih, If 2 years = 720 days and last_play_at is `less than` 730 days, would that not make it younger than 2 years?